docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  db:
+    image: mdillon/postgis
+  web:
+    build: .
+    command: bundle exec rails s -p 3000 -b '0.0.0.0'
```

Comment:
I found replacing this with 
`command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"`
got around an issue I found where rails refused to boot if it found a `server.pid` file:
```
web_1  | A server is already running. Check /myapp/tmp/pids/server.pid.
web_1  | Exiting
```